### PR TITLE
Add OWNERS file

### PR DESCRIPTION
**Pull request by nikhita**
Prow has been enabled on this repo, so we need to have an OWNERS file.

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+reviewers:
+  - alvaroaleman
+  - mrIncompetent
```
Comment (mrIncompetent) [inline OWNERS]: ~mrIncompetent~

File: OWNERS [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  - nikhita
+approvers:
+  - alvaroaleman
+  - mrIncompetent
```
Comment (mrIncompetent) [inline OWNERS]: `mrIncompetent` -> `nikhita`

**Review by mrIncompetent — CHANGES_REQUESTED** [review]

File: OWNERS (hunk 2 above)
Comment (nikhita) [reply OWNERS]: Done.

File: OWNERS (hunk 1 above)
Comment (nikhita) [reply OWNERS]: Done.

**Review by alvaroaleman — APPROVED** [review]

**Review by mrIncompetent — APPROVED** [review]

**Comment by nikhita** [comment]
Why isn't the prowjob working though O_o

**Comment by alvaroaleman** [comment]
Hm, this may be a chicken-egg problem as we have no approvers yet...

/approve

**Comment by nikhita** [comment]
Prowjob succeeded. \o/

Manually adding the approved label to ensure that Tide can merge.

**Pull request merged**